ide/editor/src/org/netbeans/modules/editor/NbEditorToolBar.java

```diff
@@ -200,6 +200,9 @@ public NbEditorToolBar(JTextComponent component) {
         
         refreshToolbarButtons();
         setBorderPainted(true);
+
+        // Style editor toolbar separately from main toolbar [style].nb-editor-toolbar   = background: #FFFFFF
+        putClientProperty("FlatLaf.styleClass", "nb-editor-toolbar");
```

Comment:
@DevCharly could this be put into `o.n.swing.laf.flatlaf` somehow? e.g `FlatLFCustoms` or similar?